layouts/shortcodes/docs_index.html

```diff
@@ -35,6 +35,7 @@ <h1 id="{{ anchorize (i18n "advanced_subscriber_information") }}">{{ i18n "advan
     <li>{{ template "link" (dict "context" . "page" "/docs/ipv6") }}</li>
     <li>{{ template "link" (dict "context" . "page" "/docs/a-warm-welcome-to-asn1-and-der") }}</li>
     <li><a href="/2019/10/09/onboarding-your-customers-with-lets-encrypt-and-acme.html">{{ i18n "onboarding_customers" }}</a></li>
+    <li>{{ template "link" (dict "context" . "page" "/docs/lencr.org") }}</li>
```

Comment:
```suggestion
    <li>{{ template "link" (dict "context" . "page" "/docs/lencr-org.md") }}</li>
```

Once you have created all other files with
```
---
title: lencr.org
slug: lencr.org
untranslated: 1
---

```

It should work.

I think the dot in the file slug make guho confused, with the file name it works (and the generated link is correctly `/docs/lencr.org/`)